src/promises/step-06.js

```diff
@@ -15,9 +15,9 @@ const countLines = (text) => {
   return text.split('\n').length
 }
 
-glob(`${srcDir}/**/*.txt`)
+glob(`${srcDir}/*.txt`)
```

Comment:
This change means that the `glob` only matches files directly in `srcDir`. Using `/**/*.txt` finds all files recursively.